### PR TITLE
Use Disruption rather than impact for traffic_reports

### PR DESCRIPTION
**Pull request by vincentlepot**
# Description

This PR addresses performance issues on /traffic_reports API since latest optimizations

## Pull Request type (optional)

This is a performance optimization

## How to test

Here are the following steps to test this pull request:

- /traffic_reports should act the same way as before but faster

## Team reviewers

BOT team

### CODE REVIEW

**Comment by kadhikari** [comment]
Some tests in the file features/traffic_reports.feature fail in my machine. I will post corrections

File: tests/features/traffic_reports.feature [hunk 1]
```diff
@@ -775,19 +775,13 @@ Feature: traffic report api
...
+        And the field "disruptions" should contain all of "{"status": "active", "id": "7ffab232-3d47-4eea-aa2c-22f8680230b7", "disruption_id": "7ffab230-3d48-4eea-aa2c-22f8680230b7"}"
         And the field "traffic_reports.0.network.id" should be "network:JDR:1"
         And the field "traffic_reports.0.network.name" should be "RATP"
         And the field "traffic_reports.0.network.links" should have a size of 2
-        And the field "traffic_reports.0.network.links.0.id" should be "7ffab232-3d47-4eea-aa2c-22f8680230b6"
-        And the field "traffic_reports.0.network.links.0.rel" should be "disruptions"
-        And the field "traffic_reports.0.network.links.1.id" should be "7ffab232-3d47-4eea-aa2c-22f8680230b7"
-        And the field "traffic_reports.0.network.links.1.rel" should be "disruptions"
+        And the field "traffic_reports.0.network.links" should contain all of "{"id": "7ffab232-3d47-4eea-aa2c-22f8680230b6", "rel": "disruptions"}"
+        And the field "traffic_reports.0.network.links" should contain all of "{"id": "7ffab232-3d47-4eea-aa2c-22f8680230b7", "rel": "disruptions"}"
```
Comment (azime) [inline tests/features/traffic_reports.feature]: can you add one test with 2 impacts:
first impact with status=published
second impact status=archived
/traffic_reports should not have the impact archived

**Comment by kadhikari** [comment]
jenkins test this please

File: chaos/utils.py [hunk 2]
```diff
@@ -603,15 +603,17 @@ def get_traffic_report_objects(disruptions, navitia):
...
-                    manage_network(result, impact, pt_object, navitia)
-                else:
-                    if pt_object.type not in collections:
-                        logging.getLogger(__name__).debug(
-                            'PtObject ignored: {type} [{uri}], not in collections {col}'.
-                            format(type=pt_object.type, uri=pt_object.uri, col=collections)
-                        )
-                        continue
-                    manage_other_object(result, impact, pt_object, navitia, collections[pt_object.type])
+            if impact.status == 'published':
```
Comment (azime) [inline chaos/utils.py]: can you add this this in [relationship](https://github.com/vincentlepot/Chaos/blob/dedce83f8fee61602c6d0c4e115f9171ad47dd12/chaos/models.py#L439), no?

File: tests/features/traffic_reports.feature (hunk 1 above)
Comment (kadhikari) [inline tests/features/traffic_reports.feature]: done

**Comment by kadhikari** [comment]
Tests failed because navitia customer in not disponible. Devups are trying to solve this problem.

**Comment by kadhikari** [comment]
jenkins test this please

**Pull request merged**